newton/utils/import_usd.py

```diff
@@ -334,6 +334,7 @@ def parse_body(rigid_body_desc, prim, incoming_xform=None, add_body_to_builder=T
                 key=path,
                 armature=body_armature,
             )
+            builder.add_joint_free(b)
```

Comment:
I'm not sure you can insert a joint here already, have you tested this with an articulated model? The joints will be inserted after the bodies are added, so I think you need to check for bodies that have no joint after all the joints have been added. This should also only be done on nonstatic bodies (i.e., whose mass > 0). It might be even better to add a helper method to `ModelBuilder` for this.

Reply:
Thanks for catching this. I updated the logic in a new commit. Please let me know if it's still problematic.

Is there a standard example with articulated model you suggest me to test with?